README.adoc

```diff
@@ -100,8 +100,24 @@ This operator currently only supports one UnifiedPushServer CR to be
 created.
 ====
 
-There are currently no configurable fields in the UnifiedPushServer CR
-apart from the name, so the example in
+Here are all of the configurable fields in a UnifiedPushServer:
```

Comment:
![image](https://user-images.githubusercontent.com/442386/60265186-088a8180-98dd-11e9-868c-48d179f8897d.png)
^Click the rich diff button at the top right of this file to see how this will render.